pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -24,7 +24,9 @@ const (
 )
 
 func MaybeAddDomainConfig(ctx *zedmanagerContext,
-	aiConfig types.AppInstanceConfig, ns *types.AppNetworkStatus) error {
+	aiConfig types.AppInstanceConfig,
+	aiStatus *types.AppInstanceStatus,
```

Comment:
Can this function modify aiStatus? If not, by define it as such?

Reply:
Is it not better to pass an argument by reference rather than passing by value, even though the argument will not be modified? In case of big sized arguments i.e structures, which is the case here..

Reply:
Done.. made the change,.